walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -326,29 +326,27 @@ The *Integration Flow Editor* should now list your Slack connection with a *Data
 . Enter the following in the *string* field under the menu:
 +
 ----
-Hello from
+Hello from 
 ----
 
-. Click *Done* to return to the *Integration* Flow Editor. The *Integration Flow Editor* should now list your *Provided API Return Path* with a *Data Type Mismatch* warning.
+. Click *Done* to return to the *Add to Integration* screen. This screen should now list your *Provided API Return Path* with a *Data Type Mismatch* warning.
 
-. Click the *Warning Icon* and choose *Add a data mapping step*.
+. Click the *Warning Icon* on the *Provided API Return Path* panel and choose *Add a data mapping step*.
 
 . From the *Data Mapper* screen click the *Request* field in the *Source* panel to expand it.
```

Comment:
This seems unnecessary, only the `message` field is selected wich is under the Message dropdown:

![fuse](https://user-images.githubusercontent.com/1851198/70337563-a9497980-184b-11ea-9c41-d8803b1090b9.png)

Reply:
@pb82 I *think* I understand what you're saying here. Let me know if the change I pushed makes sense.

Reply:
It's ok now. My point was that you don't have to expand `Requests` at all because it's not used. Only `message` is used.